nixos/modules/config/no-x-libs.nix

```diff
@@ -42,6 +42,7 @@ with lib;
         gst-plugins-base = super.gst_all_1.gst-plugins-base.override { enableWayland = false; enableX11 = false; };
         gst-plugins-good = super.gst_all_1.gst-plugins-good.override { enableX11 = false; };
       };
+      gtk3 = super.gtk3.override { x11Support = false; waylandSupport = false; };
```

Comment:
gtk3 is one of the dependencies we try to avoid and remove in many other packages in this list. What was your motivation to add this? We should probably remove the dependency from the package itself instead. Or in addition to this.

Reply:
I couldn't build the qemu package. Dependency chain - qemu -> pipewire -> ffado -> ... -> gtk3.

Reply:
if we remove ffado from pipewire, then we don't need it the gtk3 overwrite here. Also this probably hides usages of gtk3 in the future. I think we should remove this line again.